src/plugins/cordova-plugin-battery-status/sim-host.js

```diff
@@ -1,14 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
+var telemetry = require('../../client-common/client-telemetry-helper');
```

Comment:
Great, I didn't know this; it's a much better approach. Will be in my next commit